crispy_bootstrap5/templates/bootstrap5/layout/formactions.html

```diff
@@ -1,9 +1,6 @@
-<div{% if formactions.attrs %} {{ formactions.flat_attrs }}{% endif %} class="mb-3">
-    {% if label_class %}
-        <div class="aab {{ label_class }}"></div>
-    {% endif %}
-
-    <div class="{{ field_class }}">
```

Comment:
I think we still need to add `field_class`.

Reply:
I applied all the suggestions but then thought after thought that `field_class` might not be appropriate here: Would `field_class` not be applied to each child field of `FormActions`?

Consider the following test:

```py
def test_formactions_field_class(self):
        test_form = SampleForm()
        test_form.helper = FormHelper()
        test_form.helper.form_tag = False
        test_form.helper.field_class = "field-class"
        test_form.helper.layout = Layout(
            FormActions(
                Field('email'),
            ),
        )

        print(render_crispy_form(test_form))
```

Output:

```html
<div
     
    class="mb-3  field-class" 
    > <div id="div_id_email" class="mb-3"> <label for="id_email" class="form-label requiredField">
                email<span class="asteriskField">*</span> </label> <div class="field-class"> <input type="text" name="email" maxlength="30" class="textinput textInput inputtext form-control" required id="id_email"> <div id="hint_id_email" class="form-text">Insert your email</div> </div> </div> </div>
```

As per `field.html` the `<input>` gets wrapped into a div with CSS class `field-class`.

Reply:
Is `FormActions` used to wrap `Field`s? The crispy-form [docs](https://django-crispy-forms.readthedocs.io/en/latest/layouts.html) say... 

>FormActions: It wraps fields in a <div class="form-actions">. It is usually used to wrap form’s buttons. 

With an example form as per the docs. (please forgive the lack of style on the cancel button).

Main branch:

![image](https://github.com/django-crispy-forms/crispy-bootstrap5/assets/39445562/e749bdbe-0170-45e7-aba7-cb84afc46d7c)

This patch:

![image](https://github.com/django-crispy-forms/crispy-bootstrap5/assets/39445562/91d99a66-c58a-40d6-855e-3ac3d98059f8)

This therefore looks like a visual regression to me.

Reply:
Thanks! I reverted that change and also brought back the div for `label_class` to be consistent with the other template packs.